src/Arcus.Templates.Tests.Integration/Fixture/TestConfig.cs

```diff
@@ -87,12 +88,12 @@ public DirectoryInfo GetServiceBusProjectDirectory(ServiceBusEntity entity)
         /// Gets the project directory of the Azure Functions Service Bus project based on the given <paramref name="entity"/>.
         /// </summary>
         /// <exception cref="ArgumentOutOfRangeException">Thrown when no project directory can be found for the given <paramref name="entity"/>.</exception>
-        public DirectoryInfo GetAzureFunctionsServiceBusProjectDirectory(ServiceBusEntity entity)
+        public DirectoryInfo GetAzureFunctionsServiceBusProjectDirectory(ServiceBusEntityType entity)
```

Comment:
Minor: should we rename the parameter as well ?

Reply:
Yes, done!